packages/frontend/app/services/channels/-utils/channel-factory.ts

```diff
@@ -56,7 +57,7 @@ export function buildVoteChain(voteChain: VoteChain): StandardVoteChain {
     action: voteChain.action,
     key: buildChannelMember(voteChain.key),
     previousVoteChain:
-      voteChain.previousVoteChain === undefined
+      voteChain.previousVoteChain === null
```

Comment:
for falsey comparisons, null and undefined are the same, so this'd be equiv:
```ts
!voteChain.previousVoteChain ? undefined : buildVoteChain(voteChain.previousVoteChain)
```
Also
```ts
voteChain.previousVoteChain && buildVoteChain(voteChain.previousVoteChain)
```
returns the right most thing

Reply:
I can look into that.

Reply:
Addressed in 820b499